zipkin-common/src/test/scala/com/twitter/zipkin/storage/DependencyStoreSpec.scala

```diff
@@ -103,28 +103,25 @@ abstract class DependencyStoreSpec extends JUnitSuite with Matchers {
     val two = Endpoint(127 << 24 | 2, 9410, "trace-producer-two")
     val three = Endpoint(127 << 24 | 3, 9410, "trace-producer-three")
 
-    val trace = List(
-      Span(10L, "get", 10L, None, Some(1445136539256150L), Some(1152579L), List(
```

Comment:
I will raise an PR to make the date range concern a separate one and more
explicit. This might result in another test to skip, but at least it will
isolate the topic to a specific test. Even if that test also needs to be
skipped, I hope this is progress.